hawkular-inventory-api/src/main/java/org/hawkular/inventory/api/model/Metric.java

```diff
@@ -35,6 +35,7 @@
 public final class Metric extends Entity<Metric.Blueprint, Metric.Update> {
 
     private final MetricType type;
+    private final Long interval;
```

Comment:
I'd maybe be more explicit here and actually call this `collectionInterval`. "Interval of a metric" sounds a bit ambiguous to me. I'd intuitively understand it as the min-max interval of values, not a collection interval.